releasenotes/notes/Look-for-image-in-spec-rather-than-status-95d2402b82068d1c.yaml

```diff
@@ -0,0 +1,15 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Make the auto-discovery kubelet listener look for docker image name
+    in the pod spec rather than in the pod status.
+    This fixes unexpected AD behaviors when when several pods with different
+    docker image name that are pointing to the same image digest are
+    running on the same node.
```

Comment:
can we reword this to be easier for customers to understand? we're fixing a bug that (in certain conditions) was causing not picking checks and logs for containers targeted by container-image-based autodiscovery